Xamarin.Forms.Core/VisualElement_StyleSheet.cs

```diff
@@ -6,11 +6,8 @@
 
 namespace Xamarin.Forms
 {
-	public partial class VisualElement : IStyleSelectable, IStylable
+	public partial class VisualElement : IStylable
 	{
-		IList<string> IStyleSelectable.Classes
```

Comment:
Because the classes are defined on `NavigeableElement`  I just moved the interface up to that class and removed from here to avoid confusion